src/main/scala/cognite/spark/v1/DataPointsRelation.scala

```diff
@@ -172,4 +183,52 @@ object DataPointsRelationV1 {
               newAllPoints)
         }
     }
+
+  def processDeleteRow(x: DeleteDataPointsItem): DeleteDataPointsRange = {
+    val id = (x.id, x.externalId) match {
+      case (Some(_), Some(_)) =>
+        throw new IllegalArgumentException(s"Delete row for Data Points can not contain both id and externalId (on row $x)")
```

Comment:
Elsewhere we just use the id and ignore the external id if both are set, I think it may be beneficial.